docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,80 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there have been three versions for how Mobile UCRs were sent to mobile devices:
+
+- **Version 1.0**: All reports are bundled together into a single fixture during the restore process.
+- **Version 1.5**: Both V1 and V2 are supported together. This is used to migrate from V1 without causing any breaking changes.
+- **Version 2.0**: Enhanced performance where each report has a fixture of its own.
+
+Deprecation of Mobile UCR V1 and V1.5
+-------------------------------------
+Both V1 and V1.5 have been deprecated to address the performance challenges encountered with these versions.
+
+Benefits of Upgrading to Mobile UCR V2
+--------------------------------------
+The introduction of Mobile UCR V2 offers significant performance and efficiency improvements around how UCR data is restored and synced.
+These improvements offer faster report lookups and restores, as well as a new sync delay option for each report that allows administrators to control
+how frequently report data is synced.
+
+Moving to Mobile UCR V2
+-----------------------
+- Applications that do not have any references to UCRs in forms, case lists and filters need only have their
+  Mobile UCR Version updated to V2 and the app will continue working as normal.
+- Applications with references to reports will need to have all manual references to the `commcare:reports`
+  fixture changed before updating to V2. These are most likely to be contained in reports modules,
+  but others may be in case lists.
+
+The above migration steps will need to be carried out for any old app versions that get restored. These restored
```

Comment:
Actually, I think I'm going to take this paragraph out entirely to avoid any potential confusion for the user. If users try to revert to an older app version containing v1 references then they should be prompted with an alert to migrate the app. When this happens they can simply refer to the migration guide again on how to achieve this.

>  old versions won't work anymore right?

Yeah if the app has its restore version set to 2 then the app wouldn't work anymore. We do block users from releasing apps containing v1 references, however once the app is migrated then it should be functional again.

> We have flagged that to the users/teams?

This gets flagged to the user with UI alerts. After reverting to an app version containing v1 refs, we show a prompt telling them to migrate the app to v2.

Reply:
Removed in [da3846a](https://github.com/dimagi/commcare-hq/pull/35276/commits/da3846ae0de7d0591b6923b014f4f5b1d4ffee60).